### DISCUSSION TITLE
Select interpolation different than crowdin

### DISCUSSION THREAD

**Discussion by cipriancaba** [amannn/next-intl#703] [Q&A] [answered]
### Description

As specified here:
https://next-intl-docs.vercel.app/docs/usage/messages#selecting-enum-based-values
the select translation should be:
```
"message": "{gender, select, female {She} male {He} other {They}} is online."
t('message', {gender: 'female'}); // "She is online."
```

However, trying to translate this on crowdin, you get the error, saying the enum value should be between the `{}` (this is also consistent with the other replacements). Is it possible that you implemented this the other way around?

![Screenshot 2023-12-06 at 11 42 14](https://github.com/amannn/next-intl/assets/695515/4094242a-07a0-469b-87be-8b70eb40c38d)

### Mandatory reproduction URL (CodeSandbox or GitHub repository)

https://codesandbox.io/p/sandbox/next-intl-bug-template-app-forked-zcymvq

### Reproduction description

Add an enum translation

### Expected behaviour

ENUM values should be between brackets

**Comment by amannn** [comment, marked as answer]
I think you have two syntax errors:
1. A `,` is missing after `select`
2. A closing `{` is missing at the end

Fixed version:

```
{value, select, ANNOUNCEMENT {Announcement} DECISION {Decision} DOCUMENT {Document} TICKET {Ticket} other {Other}}
```

With that, your string works for me:

<img width="899" alt="image" src="https://github.com/amannn/next-intl/assets/4038316/1bb888fc-4b23-4a4e-baeb-344a0fdce206">

I'll move this to a discussion since it seems to be a usage question.

**Reply by cipriancaba** [reply]
Damn, that makes so much more sense :) Been editing multiple rows at a time and it seems I messed up all my enums except for the first one
